backend/src/js/filter.js

```diff
@@ -58,6 +58,7 @@ function filterByPrice(e) {
 
 
     if (allProductsBtn.includes("All")) {
+
```

Comment:
remove this empty line and have a better recheck of this point through all your project